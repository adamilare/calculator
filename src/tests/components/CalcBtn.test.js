import React from 'react';
import renderer from 'react-test-renderer';

import CalcBtn from '../../components/CalcBtn'; // eslint-disable-line

test("Calculator's buttons display correctly", () => {
  const onClick = jest.fn();
  const tree = renderer.create(<CalcBtn onClick={onClick} />).toJSON();
  expect(tree).toMatchSnapshot();
});
