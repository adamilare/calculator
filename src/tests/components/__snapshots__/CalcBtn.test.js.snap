// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator's buttons display correctly 1`] = `
<button
  className="calc-btn "
  onClick={[Function]}
  type="button"
/>
`;
